api/preprints/serializers.py

```diff
@@ -287,6 +291,16 @@ def update(self, preprint, validated_data):
             preprint.original_publication_date = validated_data['original_publication_date'] or None
             save_preprint = True
 
+        if 'conflict_of_interest_statement' in validated_data:
+            if flag_is_active(self.context['request'], SLOAN_COI):
```

Comment:
I don't believe this is correct. They should be able to edit the COI regardless of the flag being active for the user, they just wouldn't be able to see the results if they don't have the flag active. But this makes me realize that we don't have any waffle switches for the functionality as a whole. More on this below.